dali/dalidiag/dalidiag.cpp

```diff
@@ -302,6 +302,7 @@ void partInfo(const char *name,unsigned copy)
 
 void nodeInfo(const char *ip)
 {
+#ifndef _CONTAINERIZED
```

Comment:
minor: perhaps should throw an unexpected error, since it should never be called in container mode